lib/tilex/notifications/notifiers_supervisor.ex

```diff
@@ -4,10 +4,18 @@ defmodule Tilex.Notifications.NotifiersSupervisor do
 
   use Supervisor
 
-  def start_link() do
+  def start_link([]) do
     Supervisor.start_link(__MODULE__, :ok, name: __MODULE__)
   end
 
+  def child_spec(opts) do
```

Comment:
hey @tmock12 I was checking this code and this got me curious about that. I end up writing this as a TIL, so basically this function here is exactly the same the one generated when we invoke `use Supervisor`, exactly the same, so I wonder if you think that we should get rid of that or if you think that's worth it to keep it to be explicit?